README.md

```diff
@@ -113,12 +113,12 @@ We would like you to complete the following tasks:
 ## Submission Guidance
 
 ### Shoulds
-- Only use plain Terraform in your solution
+- Use only plain Terraform in your solution, i.e. anything supported by the Terraform CLI installed by the `run.sh` script, but not external tooling or libraries that wrap or extend Terraform, such as Terragrunt or tfenv
```

Comment:
We could add terragrunt and other file extensions into .gitignore and leave a comment in the .gitignore saying this is required. This way, if somebody tries to check it in, they'll see their files being omitted, they will open the .gitignore and see why

Reply:
I think this would be a really nice addition if this were an internal repository, to help prevent accidental inclusion of files we think could be included but don't want to be, but for the purposes of this technical task I'd personally rather leave this just in the documentation.  We've made it clear with both "should" and "should not" items, so if a candidate ends up submitting a solution which relies on additional tooling then it tells us they didn't read the assignment properly.